programs/editor/wodotexteditor.js

```diff
@@ -298,6 +301,7 @@ var Wodo = Wodo || (function () {
             //
             eventNotifier = new core.EventNotifier([
                 EVENT_UNKNOWNERROR,
+                EVENT_MODIFIEDCHANGED,
```

Comment:
How about "EVENT_ISMODIFIED_CHANGED"? The underscores help delineate the phrase a little better... ISMODIFIED seems like a consistent concept with the API you described...?